spec/dockly/docker_spec.rb

```diff
@@ -148,7 +148,7 @@
   end
 
   describe "#export_image", :docker do
-    let(:image) { Docker::Image.create('fromImage' => 'base') }
+    let(:image) { Docker::Image.create('fromImage' => 'ubuntu') }
```

Comment:
@nahiluhmot Can you scope this to a specific tag of the ubuntu image? Preferably an LTS like 14.04?